ktor-server/ktor-server-core/jvm/src/io/ktor/response/ResponseTypeAttributeKey.kt

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.response
+
+import io.ktor.util.*
+import kotlin.reflect.*
+
+public val ResponseTypeAttributeKey: AttributeKey<KType> = AttributeKey("ResponseTypeAttributeKey")
```

Comment:
Do we need it public?

Reply:
Yes, because we write with this key in `ktor-server-core` but read in `ktor-serialization`

Reply:
We could make it `@InternalAPI`